### PR TITLE
Allow claim action for every logged in person

### LINKED ISSUE #2647: Example: Unable to log in with MetaMask wallet
## Steps to reproduce

1. Send tokens to a colony that you don't have permissions in

## Expected behaviour

* Should be able to claim tokens for the colony

## Actual behaviour

* Can't claim tokens
image

### PR DESCRIPTION
**Pull request by alicjakujawa**
## Description

This PR updates logic for claim button. 

* Every logged in person can claim tokens

Resolves #2647

### CODE REVIEW

**Review by ArmandoGraterol — APPROVED** [review]
It's working as expected! Good work

File: src/modules/dashboard/components/UnclaimedTransfers/UnclaimedTransfersItem.tsx [hunk 1]
```diff
@@ -136,7 +126,7 @@ const UnclaimedTransfersItem = ({
           error={ActionTypes.COLONY_CLAIM_TOKEN_ERROR}
           success={ActionTypes.COLONY_CLAIM_TOKEN_SUCCESS}
           transform={transform}
-          disabled={!userHasPermission || !isNetworkAllowed}
+          disabled={!isNetworkAllowed || ethereal}
```
Comment (rdig) [inline src/modules/dashboard/components/UnclaimedTransfers/UnclaimedTransfersItem.tsx:129]: So this needs to check if the user has a registered profile as well, not just that it's not an ethereal wallet.

Something like this:
```js
  const { walletAddress, username, ethereal } = useLoggedInUser();

  const hasRegisteredProfile = !!username && !ethereal;
```

This is not necessarily something in the contracts, but it something we don't allow in the dapp, if you don't have your profile registered first.

**Review by rdig — APPROVED** [review]
Other than change I pointed out, this is good to go

**Review by chinins — APPROVED** [review]
Good job!

**Pull request merged**